.buildkite/run_functional_test.sh

```diff
@@ -10,21 +10,22 @@ if [ "$MACHINE_TYPE" != "x86_64" ] && [ -v SKIP_AARCH64 ]; then
   exit
 fi
 
+source ~/.bash_profile
+
+pyenv global $PYTHON_VERSION
+echo "Python version:"
+pyenv global
 
 BASEDIR=$(realpath $(dirname $0))
 ROOT=$(realpath $BASEDIR/../)
 
-# TODO to be moved in the image at https://github.com/elastic/ci-agent-images/blob/main/vm-images/enterprise-search/scripts/connectors-python/install-deps.sh#L6
-sudo apt-get -y install liblz4-dev libunwind-dev
```

Comment:
This was inlined into the VM image here: https://github.com/elastic/ci-agent-images/pull/828/files#diff-6942f445af31cff91a64513380d85c7ae18e5973d4d91ec6171cc0d9e831b7b0R11-R12